test3.tf

```diff
@@ -0,0 +1,131 @@
+      read                  = false
+      write                 = true
+      version               = "1.0"
+      retention_policy_days = 10
+    }
+    hour_metrics {
+      enabled               = true
+      include_apis          = true
+      version               = "1.0"
+      retention_policy_days = 10
+    }
+    minute_metrics {
+      enabled               = true
+      include_apis          = true
+      version               = "1.0"
+      retention_policy_days = 10
+    }
+  }
+}
+
+resource "azurerm_sql_server" "example" {
+  name                         = "sqlserver-${random_integer.rnd_int.result}"
+  resource_group_name          = "testrg"
+  location                     = "East US"
+  version                      = "12.0"
+  administrator_login          = "ariel"
+  administrator_login_password = "Aa12345678"
+}
+
+resource "azurerm_mssql_server_security_alert_policy" "example" {
+  resource_group_name        = "testrg"
+  server_name                = azurerm_sql_server.example.name
+  state                      = "Enabled"
+  storage_endpoint           = azurerm_storage_account.example.primary_blob_endpoint
+  storage_account_access_key = azurerm_storage_account.example.primary_access_key
+  disabled_alerts = [
+    "Sql_Injection",
+    "Data_Exfiltration"
+  ]
+  retention_days = 20
+}
+
+resource "azurerm_mysql_server" "example" {
+  name                = "mysql-${random_integer.rnd_int.result}"
+  location            = "East US"
+  resource_group_name = "testrg"
+
+  administrator_login          = "FGdfgbv"
+  administrator_login_password = random_string.password.result
+
+  sku_name   = "GP_Gen5_2"
+  storage_mb = 5120
+  version    = "5.7"
+
+  auto_grow_enabled                 = true
+  backup_retention_days             = 7
+  infrastructure_encryption_enabled = true
+  public_network_access_enabled     = true
+  ssl_enforcement_enabled           = false
+}
+
+resource "azurerm_postgresql_server" "example" {
+  name                         = "postgresql-${random_integer.rnd_int.result}"
+  location                     = "East US"
+  resource_group_name          = "testrg"
+  sku_name                     = "B_Gen5_2"
+  storage_mb                   = 5120
+  backup_retention_days        = 7
+  geo_redundant_backup_enabled = false
+  auto_grow_enabled            = true
+  administrator_login          = "DKFJnisdfu"
+  administrator_login_password = "Aa12345678"
+  version                      = "9.5"
+  ssl_enforcement_enabled      = true
+}
+
+resource "azurerm_postgresql_configuration" "thrtottling_config" {
+  name                = "connection_throttling"
+  resource_group_name = "testrg"
+  server_name         = azurerm_postgresql_server.example.name
+  value               = "off"
+}
+
+resource "azurerm_postgresql_configuration" "example" {
+  name                = "log_checkpoints"
+  resource_group_name = "testrg"
+  server_name         = azurerm_postgresql_server.example.name
+  value               = "off"
```

Comment:
:exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure server parameter 'log_checkpoints' is set to 'ON' for PostgreSQL Databa'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_fix_this_violation** | Fix this violation via a commit to this branch |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

 <!-- a860d86b-bb03-4371-9b57-cca914ea589d -->